text/0032-access_command.md

```diff
@@ -0,0 +1,215 @@
+```
+
+Remotes, on the other hand _can_ accept unpublished datasets. Remote configuration will get a `requirePublish` configuration parameter to get the same registry-like behaviour. Pushing unpublished datasets is aimed at the collaborator use-case, where a datasets are explicitly pushed to trusted colleagues. when a user is pushed an unpublished dataset, it will show up when they run `qri list` locally, but this dataset won't be broadcast back to the network.
+
+† _encrypted datasets don't exist yet._
+
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+### The `Published` field
+For some time we've modelled `published` as a boolean field outside the dataset itself, stored as a logbook operation. We've struggled with this field not capturing the problem of _where_ a dataset's been published. Re-conceiving of `published` as a "permission bit" clarifies the intent of this field. Published now answers "can the existence of this dataset be broadcasted" for the _entire history_ of a dataset. 
+
+### Corner cases
+A few extra bullet points to consider while we're implementing this:
+
+* qri list `--published` should work as it does today, showing only local published datasets
+* We should create a new set of `lib.AccessMethods`, talking to a number of subsystems for access control management.
+* many `publish` tests should move over to the new `push` command
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+Access control is really irritating.
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+### `listed` and `unlisted`
+Instead of the publish/unpublish terminology, we could use a more direct reference to list visibility. We can't use `list` directly because `qri access list` should be the command for listing all access-controlled datasets (if such a thing turns out to be necessary). An alternative would be to use a different conjucation or tense for list like "listed" & "unlisted". using a past tense is the best I could come up with, and in my opinion `$ qri access listed` isn't very memorable, and feels like your asking about things that have been listed in the past?
+
+The other thing to keep in mind: unlisted datasets shouldn't show up in feeds. The registry requires "published: true", but feeds built on the peer-2-peer network won't. Peers building feeds should only use datasets that are published. This clearly aligns
+
+# Prior art
+[prior-art]: #prior-art
+
+* Google Docs Sharing settings
+* Unlisted Vimeo Videos
+* RBAC - Role Based Access Control
+
+# Unresolved questions
+[unresolved-questions]: #unresolved-questions
+
+### An "Access" Data Structure
+This RFC describes `published` as a _permissions flag_. At some point we should be looking to build a data structure that describes permissions. `Published` might be a field on that data structure. The challenge here: permissions will apply to, what, users? groups? Needs more thought.
```

Comment:
Or is this the eventual answer to my question above? 🤔

Reply:
I don't think it is. In my mind this permissions structure is independent of where a thing is replicated.